Automattic/wp-calypso
# Add extra description for Jetpack AI tiered plans

This commit adds a a new inline description for Jetpack AI in case it's a tiered volume plan. This is going to be used for the new tiered AI plans.



## Proposed Changes

* This PR adds a condition to `getLabel` to display an extra description for Jetpack AI tiered plans. The goal is to make it clear to the user which plan they are purchasing.


Before | After
---|---
<img width="633" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/11dbb523-86e8-4f3a-afb1-3abc97557390"> | <img width="642" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/aada203f-f217-49ce-925c-0188b7710f25">
<img width="1062" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/595da2df-8128-45e7-be90-a1a1015a38bd"> | <img width="1062" alt="image" src="https://github.com/Automattic/wp-calypso/assets/5714212/1850807b-22fe-4d69-9f3a-98a8680be781">

## Testing Instructions



* Visit the Calypso live link
* Go to /checkout/jetpack_ai_monthly and confirm nothing changes
* Now, follow the instructions in D126642-code
* Visit /checkout/jetpack_ai_monthly:-q-100 and confirm the number of requests is part of the product title
* Purchase the plan using a test card or credits
* Go to /me/purchases, click to see the new purchase
* Confirm that you see something like "AI (100 requests per month)"

## Pre-merge Checklist
